src/main/java/net/seesharpsoft/intellij/plugins/csv/editor/table/swing/CsvTableEditorSwing.java

```diff
@@ -255,6 +266,9 @@ protected void afterTableComponentUpdate(Object[][] values) {
         try {
             DefaultTableModel tableModel = this.getTableModel();
             tableModel.setColumnIdentifiers(generateColumnIdentifiers(values, tableModel.getColumnCount()));
+            int baseFontSize = getGlobalFontSize();
+            setFontSize(baseFontSize);
```

Comment:
now you reset the font-size with every interaction like adding row/column, undo/redo etc.

the initial font-size initialization belongs to `initializedUIComponents`